python/storage-initializer.Dockerfile

```diff
@@ -7,13 +7,21 @@ FROM ${BASE_IMAGE} AS builder
 # Install Poetry
 ARG POETRY_HOME=/opt/poetry
 ARG POETRY_VERSION=1.8.3
+ARG CARGO_HOME=/opt/.cargo/
 
 # Required for building packages for arm64 arch
-RUN apt-get update && apt-get install -y --no-install-recommends python3-dev build-essential && apt-get clean && \
+RUN apt-get update && apt-get install -y --no-install-recommends python3-dev build-essential && \
```

Comment:
Have tested it ?

Reply:
@sivanantha321  I have rebuilt image with the recent changes and the build was successful . Please let me know if there are any additional tests needed.